### PR TITLE
Add Command Response. And remove Rocketload check. product and catego…

### PR DESCRIPTION
**Pull request by nawalhof**
…ry check optimize.

### CODE REVIEW

File: src/Savvii/CheckPerformanceCommand.php [hunk 1]
```diff
@@ -20,9 +20,10 @@
 use Savvii\CheckPerformanceRows\MoveScriptRow;
 use Savvii\CheckPerformanceRows\PagespeedRows;
 use Savvii\CheckPerformanceRows\MySQLTableSizeRows;
-use Savvii\CheckPerformanceRows\RocketLoaderRow;
+//use Savvii\CheckPerformanceRows\RocketLoaderRow;
```
Comment (toonvd) [inline src/Savvii/CheckPerformanceCommand.php]: A complete remove / cleanup is allowed here. Rocket loader does not contribute to lighthouse scores anymore.

File: src/Savvii/CheckPerformanceRows/PagespeedRows.php [hunk 2]
```diff
@@ -83,14 +83,35 @@ public function getRow()
...
-            'Category' => $category->setStoreId($selectedStoreId)->getUrl()
-        );
 
         $result = array();
+        $pagesToCheck = [
+            'Home' => $store->getBaseUrl(),
+            'Cart' => $store->getUrl('checkout/cart', ['_secure' => true])
+        ];
+        /** Check if product has an ID */
+        if ($product->getId()) {
```
Comment (toonvd) [inline src/Savvii/CheckPerformanceRows/PagespeedRows.php:99]: Avoid if else if possible

File: src/Savvii/HelloWorldCommand.php [hunk 3]
```diff
@@ -2,9 +2,10 @@
 
 namespace Savvii;
 
-use N98\Magento\Command\AbstractMagentoCommand;
+use \N98\Magento\Command\AbstractMagentoCommand;
```
Comment (toonvd) [inline src/Savvii/HelloWorldCommand.php]: Please remove the \

File: src/Savvii/CheckPerformanceCommand.php (hunk 1 above)
Comment (nawalhof) [reply src/Savvii/CheckPerformanceCommand.php]: Fixed

File: src/Savvii/HelloWorldCommand.php (hunk 3 above)
Comment (nawalhof) [reply src/Savvii/HelloWorldCommand.php]: Fixed

File: src/Savvii/CheckPerformanceRows/PagespeedRows.php (hunk 2 above)
Comment (nawalhof) [reply src/Savvii/CheckPerformanceRows/PagespeedRows.php:99]: Key of the array is overwritten if product is tested.
Refactor the else away

**Pull request merged**